conference/src/main/java/com/kyriba/conference/management/api/ScheduleController.java

```diff
@@ -92,15 +92,15 @@ void removePresentation(
   @ApiOperation(value = "Change presentations parameters")
   @PutMapping(value = "/presentations/{id}", consumes = APPLICATION_JSON_UTF8_VALUE)
   void updatePresentation(
-      @Positive @ApiParam(value = "Presentation identity", required = true) @PathVariable long id,
-      @ApiParam(value = "Presentation change object", required = true) @RequestBody PresentationRequest presentationRequest)
+      @ApiParam(value = "Presentation identity", required = true) @PathVariable long id,
+      @Valid @ApiParam(value = "Presentation change object", required = true) @RequestBody PresentationRequest presentationRequest)
   {
     scheduleService.updatePresentation(id, presentationRequest);
   }
 
 
   @ExceptionHandler({ LinkedEntityNotFoundException.class, InvalidPresentationTimeException.class,
-      PresentationTimeIntersectionException.class })
+      PresentationTimeIntersectionException.class, ConstraintViolationException.class })
   void handleException(HttpServletResponse response, RuntimeException e) throws IOException
```

Comment:
Things like validation are common for all controllers, so defined in dedicated class or base class for all controllers.

Reply:
done